tools/README.md

```diff
@@ -2,173 +2,176 @@
 
 _NB: this folder is yarn only_
 
-## Moonbeam-Launch
+## Launching complete network
 
-_Moonbeam-launch_ (https://github.com/PureStake/polkadot-launch/tree/moonbeam-launch) is a fork of polkadot-launch adapted for Moonbeam's needs for testing. Eventually, polkadot-launch should catch up and remove the need for a custom implementation.
+Based on [polkadot-launch](https://github.com/paritytech/polkadot-launch), the tool to launch
+multiple relay and parachain nodes, the script [launch.ts](./launch.ts) allows to start a complete
+network based on the different version of the runtimes
 
-_Polkadot-launch_ (https://github.com/paritytech/polkadot-launch) allows to start a local network of polkadot relaychain and parachain nodes with the desired configuration.
+As the moonbeam and relay runtimes evolved, more configurations will be added to the script.
 
-For this setup, you want to have moonbeam and polkadot cloned in the same repo:
+To make it easier and faster to run, it will detect and download the binaries
+from the given docker images.  
+(This is only supported on Linux. Other OS must use local configuration, see further)
 
-- repo
-  - moonbeam
-  - polkadot
+### Installation
 
-### Build Parachain
+```
+yarn install
+```
 
-In the moonbeam repo, checkout to the desired commit of moonbeam and then run:
+### Usage
 
 ```
-cargo build --release
-./target/release/moonbeam --version
+yarn run launch --parachain alphanet-8.1
```

Comment:
I ran this and got 
```
yarn run launch --parachain alphanet-8.1
yarn run v1.22.10
warning package.json: No license field
$ ts-node launch --parachain alphanet-8.1
🚀 Relay:     rococo-9004          - purestake/moonbase-relay-testnet:sha-2f28561a (rococo-local)
     Missing build/alphanet-8.1/moonbeam locally, downloading it...
Unable to find image 'purestake/moonbeam:v0.8.1' locally
v0.8.1: Pulling from purestake/moonbeam
69692152171a: Already exists
6c7f24544c45: Pulling fs layer
6bcfc52cc170: Pulling fs layer
e594c340c5f2: Pulling fs layer
6c7f24544c45: Download complete
6c7f24544c45: Pull complete
6bcfc52cc170: Download complete
e594c340c5f2: Verifying Checksum
e594c340c5f2: Download complete
6bcfc52cc170: Pull complete
e594c340c5f2: Pull complete
Digest: sha256:ed682ab6de6005486b11d67c8ba4addec0564e8bdb7d530fb91f101359c91320
Status: Downloaded newer image for purestake/moonbeam:v0.8.1
     build/alphanet-8.1/moonbeam downloaded !
🚀 Parachain: alphanet-8.1         - purestake/moonbeam:v0.8.1 (moonbase-local)

/Users/antoineestienne/GitHubRepo/double_repos/moonbeam/tools/build/rococo-9004/polkadot: /Users/antoineestienne/GitHubRepo/double_repos/moonbeam/tools/build/rococo-9004/polkadot: cannot execute binary file
failed to parse the chain spec

Killing all processes...
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
Yes, it is not working for Mac. I added a check for that in the code. Can you try to re-run with the latest version? It should give you an error message.

Reply:
You can test it on mac with 
```
yarn run launch --parachain local --relay local
```
It will use your moonbeam and polkadot local binary (supposedly already compiled and from the same root directory)

Reply:
I had to `nvm use node` because `node` on `ubuntu` is seriously out of date. It might be worth mentioning this...

Reply:
Also getting this when trying to run: `launch.ts:17:30 - error TS2307: Cannot find module 'polkadot-launch' or its corresponding type declarations.`

Reply:
Missed this output earlier, this explains why `polkadot-launch` is missing: `error Can't add "polkadot-launch": invalid package version undefined.`

Reply:
This happens when you run `yarn` ?